linkcheck/checker/httpurl.py

```diff
@@ -111,7 +110,7 @@ def add_size_info(self):
             # Note that content-encoding causes size differences since
             # the content data is always decoded.
             try:
-                self.size = int(self.getheader("Content-Length"))
+                self.size = int(self.headers.get("Content-Length"))
             except (ValueError, OverflowError):
```

Comment:
int(None) will raise a TypeError.

I don't think you can get an OverflowError from int() in Python 3.

(Preexisting bug, fix, if any, belongs to a separate PR.)

Reply:
Ok, not a bug because this is inside an `if "Content-Length" in self.headers:`, but in that case do not use self.headers.get, use `self.headers['Content-Length']`.

Reply:
Done.